frontend/src/components/vehicle/VehicleGeneralData.tsx

```diff
@@ -0,0 +1,63 @@
+import { getVehicleType } from '@/hooks/useDetailsForVehicleType'
+import { Vehicle } from '@green-ecolution/backend-client'
+
+interface VehicleGeneralData {
+  vehicle?: Vehicle
+}
+
+const VehicleGeneralData: React.FC<VehicleGeneralData> = ({ vehicle }) => {
+  const vehicleType = getVehicleType(vehicle.type)
+
+  const vehicleData = [
+    {
+      label: 'Modell',
+      value: vehicle?.model ?? 'Keine Angabe',
+    },
+    {
+      label: 'Fahrzeug-Typ',
+      value: vehicleType ?? 'Keine Angabe',
+    },
+    {
+      label: 'Höhe des Fahrzeugs',
+      value:
+        vehicle?.height != null ? `${vehicle.height} Meter` : 'Keine Angabe',

```

Comment:
`vehicle?.height ?? 'Keine Angabe'`

Reply:
Then it would just be "Höhe des Fahrzeugs: 2" and not "Höhe des Fahrzeugs: 2 Meter" or should I do this differently somehow? Same with the others